forte/data/ontology/top.py

```diff
@@ -12,7 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from dataclasses import dataclass
+from email.mime import image
 from functools import total_ordering
+from multiprocessing.sharedctypes import Value
```

Comment:
Some unnecessary imports comes back again, please check the auto imports if you are using them